pkgs/by-name/ei/eigenlayer-cli/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGo121Module,
+  fetchFromGitHub,
+}:
+buildGo121Module rec {
+  pname = "eigenlayer-cli";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "NethermindEth";
+    repo = "eigenlayer";
+    rev = "v${version}";
+    hash = "sha256-VO56mMkUVeZN+YJ/cxPyZQ5dGvVDirslH4jhV5hpufQ=";
+  };
+
+  vendorHash = "sha256-DDGMprfryWj9Td4PX/j5Fsjm+bGGiDgnCXmm2IhRSMo=";
+
+  ldflags = [ "-s" "-w" ];
+  subPackages = ["cmd/eigenlayer"];
```

Comment:
```suggestion
  subPackages = [ "cmd/eigenlayer" ];
```

Reply:
Added extra spaces (but interesting, cuz my formatter removes them :)

Reply:
Yeah, I've seen both approaches and my formatters add them, so maybe not that important.